tap_suiteql/client.py

```diff
@@ -161,3 +164,45 @@ def post_process(self, row: dict, context: Optional[dict] = None) -> dict:
         """
 
         return row
+
+
+class QueryBuilder:
+    def __init__(self, stream: suiteqlStream):
+        self.stream: suiteqlStream = stream
+
+    def _query_builder(
+        self, schema: dict, replication_key: str, entity_name: str, stream_type: str
+    ) -> str:
+        select_statement = "select "
+        from_statement = f"from {entity_name}"
+        where_statement = "where "
+        column_select = []
+        where_clauses = ["1=1"]
+        for attribute_name, properties in schema["properties"].items():
```

Comment:
what do you think about extract this logic to another method to improve the readability of this method?

Reply:
great idea!